lib/wise_homex/models/admin_data.ex

```diff
@@ -0,0 +1,11 @@
+defmodule WiseHomex.AdminData do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :account, WiseHomex.Account
```

Comment:
You would probably also need the opposite relation, right?

Reply:
Ah yeah. I will add that!

Reply:
Fixed in https://github.com/wise-home/wise_homex/pull/221/commits/a7b590ad90b78d872808172528b23c3117a12be4